signer/src/utxo.rs

```diff
@@ -359,6 +362,54 @@ impl DepositRequest {
     }
 }
 
+impl TryFrom<model::DepositRequest> for DepositRequest {
+    type Error = Error;
+
+    fn try_from(request: model::DepositRequest) -> Result<Self, Self::Error> {
+        let txid = bitcoin::Txid::from_byte_array(
+            request.txid.try_into().map_err(|_| Error::TypeConversion)?,
+        );
+
+        let vout = request
+            .output_index
+            .try_into()
+            .map_err(|_| Error::TypeConversion)?;
+
+        let outpoint = OutPoint { txid, vout };
+
+        let max_fee = request
+            .max_fee
+            .try_into()
+            .map_err(|_| Error::TypeConversion)?;
+
+        let signer_bitmap = Vec::new(); // TODO(326): Populate
+
+        let amount = request
+            .amount
+            .try_into()
+            .map_err(|_| Error::TypeConversion)?;
+
+        let deposit_script = bitcoin::consensus::deserialize(&request.spend_script)
+            .map_err(Error::BitcoinSerialization)?;
+
+        let redeem_script = bitcoin::consensus::deserialize(&request.reclaim_script)
+            .map_err(Error::BitcoinSerialization)?;
```

Comment:
Right, yeah that's probably better. For some reason, deserialize felt more handy here. I don't remember why, because for other types I've used the direct byte conversion. I'll update.

Reply:
Done.